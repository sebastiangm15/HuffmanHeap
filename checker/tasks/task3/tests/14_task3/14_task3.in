20
1 L1
2 L2
3 L3
4 L4
5 L5
6 L6
7 L7
8 L8
9 L9
10 L10
11 L11
12 L12
13 L13
14 L14
15 L15
16 L16
17 L17
18 L18
19 L19
20 L20
10
L3
L5
L8
L10
L12
L13
L14
L15
L17
L19
