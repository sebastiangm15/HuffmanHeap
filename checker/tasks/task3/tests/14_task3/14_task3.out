01110111110110111010001101000100110101101000
